src/v0/destinations/am/transform.js

```diff
@@ -22,13 +22,13 @@ const {
   getFieldValueFromMessage,
   getValueFromMessage,
   deleteObjectProperty,
-  getErrorRespEvents,
```

Comment:
There are too many other destinations using `getErrorRespEvents` --> So all of those are also impacted ?

Reply:
Yes

Reply:
Yes

Reply:
https://linear.app/rudderstack/issue/INT-1579/fix-all-destinations-using-geterrorrespevents-to-get-right-metadata